### PR TITLE
Gimbap author

### PR DESCRIPTION
**Pull request by uplne**
Gimbap can now get author picture.

IE9:
screen shot 2016-03-29 at 15 01 12

Modern:
screen shot 2016-03-29 at 15 01 25
screen shot 2016-03-29 at 15 01 33
screen shot 2016-03-29 at 15 11 47

### CODE REVIEW

**Comment by sammorrisdesign** [comment]
Might be unrelated but why does the position of the logo change based on size of card?

**Comment by uplne** [comment]
@sammorrisdesign if you mean the vertical position I don't know. Nice catch. I had to break it somewhere in the process and got totally blind to it.

File: docs/styleguide/assets/css/styleguide.css [hunk 1]
```diff
@@ -0,0 +1 @@
...r:#000000;background-color:#ffdddd}.ge{color:#000000;font-style:italic}.gr{color:#aa0000}.gh{color:#999999}.gi{color:#000000;background-color:#ddffdd}.go{color:#888888}.gp{color:#555555}.gs{font-weight:bold}.gu{color:#aaaaaa}.gt{color:#aa0000}.kc{color:#000000;font-weight:bold}.kd{color:#000000;font-weight:bold}.kn{color:#000000;font-weight:bold}.kp{color:#000000;font-weight:bold}.kr{color:#000000;font-weight:bold}.kt{color:#445588;font-weight:bold}.m{color:#009999}.s{color:#d01040}.na{color:#008080}.nb{color:#0086B3}.nc{color:#445588;font-weight:bold}.no{color:#008080}.nd{color:#3c5d5d;font-weight:bold}.ni{color:#800080}.ne{color:#990000;font-weight:bold}.nf{color:#990000;font-weight:bold}.nl{color:#990000;font-weight:bold}.nn{color:#555555}.nt{color:#000080}.nv{color:#008080}.ow{color:#000000;font-weight:bold}.w{color:#bbbbbb}.mf{color:#009999}.mh{color:#009999}.mi{color:#009999}.mo{color:#009999}.sb{color:#d01040}.sc{color:#d01040}.sd{color:#d01040}.s2{color:#d01040}.se{color:#d01040}.sh{color:#d01040}.si{color:#d01040}.sx{color:#d01040}.sr{color:#009926}.s1{color:#d01040}.ss{color:#990073}.bp{color:#999999}.vc{color:#008080}.vg{color:#008080}.vi{color:#008080}.il{color:#009999}
```
Comment (jbreckmckye) [inline docs/styleguide/assets/css/styleguide.css]: What is this?

**Comment by jbreckmckye** [comment]
:+1: once build passes CI (I think there are some old switches that need to be removed)

**Pull request merged**